### PR TITLE
feat: add core network deployments

### PR DESCRIPTION
**Pull request by Reinis-FRP**
**Motivation**

We've deployed OO contracts on Core mainnet that needs to be added to networks file.

**Summary**

Adds Core mainnet contract deployments.

**Details**

Verified contract state with `yarn hardhat verify-admin-xchain --network core`:

```
🌚 Verifying L2 contract state 🌚
  Registry
    - OptimisticOracle registered: ✅
    - OptimisticOracleV2 registered: ✅
    - OptimisticOracleV3 registered: ✅
    - Owned by GovernorSpoke: ✅
  Store
    - Owned by GovernorSpoke: ✅
  IdentifierWhitelist
    - Owned by GovernorSpoke: ✅
  AddressWhitelist
    - Owned by GovernorSpoke: ✅
  OptimisticOracleV3
    - Owned by GovernorSpoke: ✅
  Admin_ChildMessenger
    - Set oracleSpoke: ✅
  Finder
    - Owned by GovernorSpoke: ✅
    - Set "Store" in Finder: ✅
    - Set "IdentifierWhitelist": ✅
    - Set "CollateralWhitelist" in Finder: ✅
    - Set "Oracle" in Finder (to OracleSpoke): ✅
    - Set "OptimisticOracle" in Finder: ✅
    - Set "OptimisticOracleV2" in Finder: ✅
    - Set "OptimisticOracleV3" in Finder: ✅
    - Set "ChildMessenger" in Finder: ✅
```

Note that we still need to transfer `Admin_ChildMessenger` ownership to safe multisig once that is deployed.

Also note that this setup does not yet have corresponding messenger contract on L1 which means:
- its not possible to initiate governance vote targeting Core network contracts from L1
- dispute requests would need to be manually requested on L1 locking additional bond and they would not be resolved on `OracleHub` (though, admin multisig will still be able to push DVM resolution on `OracleSpoke` through `Admin_ChildMessenger`)

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

**Issue(s)**

Fixes https://linear.app/uma/issue/UMA-2058/contracts-deployment

### CODE REVIEW

File: packages/core/contracts/cross-chain-oracle/README.md [hunk 1]
```diff
@@ -240,10 +240,10 @@ Transfer deployed contract ownership for each of `Registry`, `Store`, `Identifie
 yarn hardhat transfer-owner --network <l2name> --contract <CONTRACT_ADDRESS> --owner <GOVERNOR_SPOKE_ADDRESS>
 ```
 
-Also transfer `GovernorSpoke` contract ownership to multisig:
+Also transfer `Admin_ChildMessenger` contract ownership to multisig:
```
Comment (Reinis-FRP) [inline packages/core/contracts/cross-chain-oracle/README.md:243]: this was wrong, as GovernorSpoke is not ownable.

Comment (mrice32) [reply packages/core/contracts/cross-chain-oracle/README.md:243]: nice catch

File: packages/core/networks/1116.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+[
+  {
+    "contractName": "Registry",
```
Comment (mrice32) [inline packages/core/networks/1116.json:3]: I went through all these, spot checked ownership/fields and checked verifications. All looked good!

**Review by mrice32 — APPROVED** [review]
LGTM!

**Pull request merged**